master/internal/api_tensorboard.go

```diff
@@ -259,8 +259,6 @@ func (a *apiServer) LaunchTensorboard(
 	}
 	expConf = schemas.WithDefaults(expConf).(expconf.ExperimentConfig)
 
-	refineArgs(spec.Config.TensorBoardArgs)
```

Comment:
should we retain functionality that allows users to specify tensorboard args directly in the command?

Reply:
Did I miss something? Doesn't `spec.Config.TesnorBoardArgs` just pull from `tensorboard_args`? I thought I was removing us not being able to specify `what ever` as a tensorboard arg. It would change it to `--what ever` ... or `["t", "foo"]` would become `--t --foo`. Is there some functionality that I am missing here?

Reply:
oh no you're right - it pulls direct from the config. I approve 👍